pkg/controller/remotemachineset/remotemachineset_controller.go

```diff
@@ -275,10 +304,37 @@ func (r *ReconcileRemoteMachineSet) Reconcile(request reconcile.Request) (reconc
 	return reconcile.Result{}, r.updatePoolStatusForMachineSets(pool, machineSets, logger)
 }
 
-func (r *ReconcileRemoteMachineSet) getRemoteMachineSets(
+func (r *ReconcileRemoteMachineSet) getMasterMachine(
 	cd *hivev1.ClusterDeployment,
 	remoteClusterAPIClient client.Client,
 	logger log.FieldLogger,
+) (*machineapi.Machine, error) {
+	remoteMachines := &machineapi.MachineList{}
+	tm := metav1.TypeMeta{}
+	tm.SetGroupVersionKind(machineapi.SchemeGroupVersion.WithKind("Machine"))
+	if err := remoteClusterAPIClient.List(
+		context.Background(),
+		remoteMachines,
+		&client.ListOptions{
+			Raw: &metav1.ListOptions{
+				TypeMeta:      tm,
```

Comment:
Just curious, why is the type meta needed here?

Reply:
I don't know. It was done that way with the list of machinesets. I planned on checking whether it was actually needed, but got all my time sucked away by digging out of dependency hell.